sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -88,14 +88,11 @@ Licensed under the MIT License.
       <version>1.5.4</version> <!-- {x-version-update;io.micrometer:micrometer-registry-graphite;external_dependency} -->
     </dependency>
 
-    <!-- Issue: We have a problem with Direct TCP with using netty-tcnative when OpenSSL 1.1 is installed
     <dependency>
       <groupId>io.netty</groupId>
-      <artifactId>netty-tcnative</artifactId>
-      <version>2.0.31.Final</version> {x-version-update;io.netty:netty-tcnative;external_dependency}
-      <classifier>linux-x86_64</classifier>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
please update azure-cosmos and remove the explicit exclusion. This library is a dependency of azure-core we don't need to have explicit dependency on it.

Reply:
Thanks Mo~  I removed the exclusions from azure-cosmos and azure-cosmos-encryption